### PR TITLE
Updated splash page

### PR DESCRIPTION
**Pull request by domdreams**
I updated the splash page html/css. I tried updating the routing too but you might have to double check my work to make sure it is right. The buttons don't currently work correctly. I may need some help fixing that.

### CODE REVIEW

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -14,12 +14,13 @@ import {
...
+
+
   {
     path: 'home',
     component: HomeComponent,
     children: [
+      { path: 'splash', component: SplashComponent,  outlet: 'sidenav'},
       { path: 'mayor', component: MayorComponent, outlet: 'sidenav' },
       { path: 'city-attorney', component: CityAttorneyComponent, outlet: 'sidenav' },
       {
```
Comment (gSoD17) [inline src/app/app-routing.module.ts:26]: Checked out your branch and seeing how to mesh the new layout together.  For this part of the app-routing module, we don't need the parent-child routing like you see in the object home path.  We can simply do just this:

`{ path: '', redirectTo: 'splash',pathMatch: 'full' },`
`{ path: 'splash', component: SplashComponent },`
`{ path: 'mayor', component: MayorComponent }`
`{ path: 'under-construction', component: UnderConstructionComponent },`

We can also omit the `outlet: 'sidenav'` part as well from each of these paths.

File: src/app/components/home/home.component.html [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <div class="header-nav">
-  <div routerLink="/splash"><img class="logo" src="../../../assets/home/VV_Logo_Reversed@2x.png" alt="Voter's Voice Initiative" width="120"
+  <div routerLinkActive="active-link" [routerLink]="[{ outlets: {sidenav: ['splash']} }]"><img class="logo" src="../../../assets/home/VV_Logo_Reversed@2x.png" alt="Voter's Voice Initiative" width="120"
```
Comment (gSoD17) [inline src/app/components/home/home.component.html:2]: With the aforementioned changes for the app routing module, the initial `routerLink="/spash"` should work.  We no longer need to do the whole outlets: {sidenav: ...} format.  All you need to do is to replace the routerLinks, without the brackets, with their corresponding routes.

And for the <router-outlet>, you can also remove the `name="sidenav"`.

File: src/app/app-routing.module.ts (hunk 1 above)
Comment (domdreams) [reply src/app/app-routing.module.ts]: @gSoD17 Pushed these changes. Check it out.

File: src/app/components/home/home.component.html (hunk 2 above)
Comment (domdreams) [reply src/app/components/home/home.component.html]: @gSoD17 Fixed this as well. Check it out.

**Pull request merged**